### ISSUE TITLE
Where do the viewing conditions for sRGB in the WCAG techniques page come from?

### ISSUE THREAD

**Issue by svgeesus** [w3c/silver#634]
(I hope this is the right repo for this comment. If not, please direct me to the correct one).

This is a request for clarification.

The [Visual contrast of text](https://www.w3.org/WAI/GL/WCAG3/2022/how-tos/visual-contrast-of-text/#develop-button) page states, on the develop tab:

> The designer's workspace is important. Variations in monitor calibration and ambient lighting will affect the final output, often in unexpected ways.

>    Use a hardware calibrator such as an XRite Display, and use software to calibrate the monitor to the sRGB standard.
>    The ambient room lighting should be about 20% of the brightness of the monitor at peak white. For a monitor with a peak white of 160 cd/m2, the ambient should be around 64 lux.
>    Put another way, setting the full screen to sRGB mid grey color #808080, the monitor should appear around the same brightness as the "overall" room lighting.
>    The color temperature of the room lighting should be 5500K, ideally with a CRI over 90.
>    Don't paint the walls 18% grey. Instead, paint the walls pure white and set the lighting level so the white walls approximate the monitor at #808080 grey.
>    And finally, avoid bright colors in the field of vision when looking at a monitor. Nearby colors can affect how you perceive the monitor content.

I certainly approve of specifying the viewing environment for testing, but would like to understand where these guidelines come from, and why they are somewhat different to the sRGB viewing conditions.

The first point, to use a hardware calibrator, is a good one although what exactly should be calibrated (whitepoint, primary chromaticities, Tone Response Curve (EOTF), surround, and ambient illuminance) could use a little more detail.In particular I assume the monitor should cover 100% or more of the sRGB gamut by volume, so that it can accurately be set to emulate sRGB.

The second point sets the ambient illuminance correctly to 64 lux, but tying this to a percentage of media white seems odd (is that where the 160 cd/m<sup>2</sup> comes from? The sRGB specification sets the media white to 80 cd/m<sup>2</sup>, although that value is rarely adhered to in practice with the advent of brighter LCD screens.

Where does the room lighting color temperature of 5500K come from, and is this intended to be a daylight simulator (D55) or some other type? Is the user expected to be adapted to this white, or to the monitor D65 white? (sRGB assumes full adaptation to monitor white).

For reference, the [viewing conditions for Display P3](https://www.color.org/chardata/rgb/DCIP3.xalter) are part of the official specification](https://www.color.org/chardata/rgb/DisplayP3.xalter) and are identical to[ those for sRGB](https://www.color.org/chardata/rgb/srgb.xalter), whose official definition is sadly behind a paywall.

> Image background (proximal field): 16 cd/m2
> Viewing surround: 4.1 cd/m2
> Ambient illuminance: 64 lux
> Adapted white point luminance: Unspecified
> Adapted white point chromaticity: Unspecified

**Comment by Myndex** [comment]
Hi Chris, @svgeesus  the most current iteration of the proposed standardized viewing environment is at the main SAPC-APCA repo, in the documentation folder: 

https://github.com/Myndex/SAPC-APCA/blob/master/documentation/StandardObserverModel.md

And also, I consider all of thie preliminary, as a starting poiint for further discussion and research. The current listted specifications are based on available research and existing standards. And the discussion tab at the main SAPC/APCA repo that might be the best place to discuss these issues, and I do consider the matter open for discussion.

### _Regarding the questions_
FWIW the current WCAG 3 draft is about a year and a half behind what is at the main SAPC-APCA repo. I do have pull request #630  here awaiting to be merged.

To answer here though, you might be interested in the SID IDMS display standards, and it's free, no paywall:

https://www.sid.org/Standards/ICDM#8271483-idms-download

As to why APPLE states P3 reference envirnment as the same as sRGB, IMO only means to me there was no one at Apple paying attention.  😳


### These are issues I consider up for discussion and importantly, further research:

The 5500 K ambient illumination is one of several specified in research, is a commonly available consumer lighting CCT, and is a standard practice here in Hollywood (to have a 5500K backlight for video color grading). Here, it may historically relate to Kodak motion picture stocks being daylight balanced for 5500K or 5600K, or the fact that the HMI lighting we use on set is typically 5600K.

That said, it's likely less than critical, as chromatic adaptation to images on self-illuminated displays "largely discards" ambient influences, being more influenced by the local/near adaptation effects. (though the brown dress illusion/controversy might indicate a useful branch of research).

In terms of peak white, Among useful references is the BasICColor Display calibration software and X-Rite display calibration software, which typically indicate display reference white at 160 cd/m², or calibrated realtive to the ambient, which in real world offices and daytime environments far exceeds 64 lux (!!) (Typically 350 lux or more).

### _Additional Background_
There is massive variation in real-world scenarios, per research and surveys of actual monitors in use.

IMO, adhering to a very old standard (sRGB) that was based in CRT technology, and that is widely disregarded today, is not instructive.

**_The IEC 61966-2-1 Reference Display_**
1. Display luminance level — 80 cd/m²
2. Display white point — D65 (x = 0.3127, y = 0.3290)
3. Display gamma 2.2

The IEC 61966-2-1 reference viewing conditions are based on **ISO 3664** and stated as:
1. Reference Background — for the background as part of the display screen, the background is 20% of the reference display luminance level
2. Reference Surround — 20% reflectance of the reference ambient illuminance level
3. Reference Proximal Field — 20% of the reflectance of the reference display luminance level
4. Reference Ambient Level 64 lux 
5. Reference Ambient White Point   D50 (x = 0,3457, y = 0,3585)

It also states "typical" ambient as 350lux, quite a bit different from the "reference".

Is it appropriate to use ISO 3664 (graphic arts) view conditions? I don't know that it is. Is it appropriate to remain fixed to 25 year old technology (CRT) and an 80nit peak white, when other standards are 100nit, 120ni, 160 nit... and modern display devices can acheive 1200 nit? Again, I can't find useful support of this basis.

Most recent/relevant research is being conducted at levels above 120 cd/m².

And it should be clear that the peak white relative to the ambient adaptation has a marked influence over the curves defining perceptual lightness, and lightness contrast, which themselves are further affected by spatial frequency of stimulus and stimulus polarity.

As output display gamma curve essentially defines "contrastyness", in conjunction with the peak white and the relative ambient adapt and near adaptation, this is a useful area for further exploration. Consider in a theater, in black-out conditions (< 5 lux), the peak screen white is a meager 54.8 cd/m² (16 foot-lamberts) and DCI is a 2.6 gamma.... but 54.8 nits is really bright when dark adapted.

An interior daylight room is around 350 lux, and research shows users setting devices >200 cd/m² (Younger users set them higher than older users, per one paper).

### Related present work:
And you might be interested to know that the relative "contrast middle" for higher spatial frequencies is not at all the "middle grey" of L*, and shifts relative to total page luminance, outside of the local stimulus/background, and not only is that shift non-linear, it is not necessarily even in the same direction:

<img width="480" alt="Mid Grey L* vs Adaptation Apr 2022 this chart demonstrates the curves for middle contrast based on various overall page luminances" src="https://user-images.githubusercontent.com/42009457/172223488-87527663-b07a-435f-b85d-52d2a4e05a74.png">

And this indicates the need for expanded research and study for some of these areas, for which I am presently negotiating funding.

Outside of this, at present, APCA results are calculated/weighted to consider the "worst case" ambient conditions (that being high light adaptation) as the most useful and universal guideline.
